contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1039,6 +1039,25 @@ $ oc adm create-error-template > errors.html
 ----
 ....
 
+* If your command contains multiple lines and uses callout annotations, you must comment out the callout(s) in the codeblock, as shown in the following example:
++
+....
+To scale based on the percent of CPU utilization, create a `HorizontalPodAutoscaler` object for an existing object:
++
+[source,terminal]
+----
+$ oc autoscale <object_type>/<name> \// <1>
```

Comment:
nit: put the comment marks closer to the callout to make it clear that it is for the callout, and not escaping the previous backslash.

```suggestion
$ oc autoscale <object_type>/<name> \ //<1>
```

Reply:
That doesn't work. ✔️